README.md

```diff
@@ -1,101 +1,48 @@
 
-### Application Maintenance
-Occasionally, new package dependencies or schema changes will necessitate
-maintenance. To non-destructively update your development stack at any time:
-1. Run `docker-compose pull`
-2. Run `docker-compose build --pull --no-cache`
-3. Run `docker-compose run --rm django ./manage.py migrate`
+Or, access the OpenAPI REST interface at [https://app.shapeworks-cloud.org/api/docs/swagger/](https://app.shapeworks-cloud.org/api/docs/swagger/).
 
-## Develop Natively (advanced)
-This configuration still uses Docker to run attached services in the background,
-but allows developers to run Python code on their native system.
+For admins, visit [https://app.shapeworks-cloud.org/admin](https://app.shapeworks-cloud.org/admin).
 
-### Initial Setup
-1. Run `docker-compose -f ./docker-compose.yml up -d`
-2. Install Python 3.8
-3. Install
-   [`psycopg2` build prerequisites](https://www.psycopg.org/docs/install.html#build-prerequisites)
-4. Create and activate a new Python virtualenv
-5. Run `pip install -e .[dev]`
-6. Run `source ./dev/export-env.sh`
-7. Run `./manage.py migrate`
-8. Run `./manage.py createsuperuser` and follow the prompts to create your own user
 
-### Run Application
-1.  Ensure `docker-compose -f ./docker-compose.yml up -d` is still active
-2. Run:
-   1. `source ./dev/export-env.sh`
-   2. `./manage.py runserver`
-3. Run in a separate terminal:
-   1. `source ./dev/export-env.sh`
-   2. `celery --app shapeworks_cloud.celery worker --loglevel INFO --without-heartbeat`
-4. When finished, run `docker-compose stop`
-
-## Remap Service Ports (optional)
-Attached services may be exposed to the host system via alternative ports. Developers who work
-on multiple software projects concurrently may find this helpful to avoid port conflicts.
-
-To do so, before running any `docker-compose` commands, set any of the environment variables:
-* `DOCKER_POSTGRES_PORT`
-* `DOCKER_RABBITMQ_PORT`
-* `DOCKER_MINIO_PORT`
-
-The Django server must be informed about the changes:
-* When running the "Develop with Docker" configuration, override the environment variables:
-  * `DJANGO_MINIO_STORAGE_MEDIA_URL`, using the port from `DOCKER_MINIO_PORT`.
-* When running the "Develop Natively" configuration, override the environment variables:
-  * `DJANGO_DATABASE_URL`, using the port from `DOCKER_POSTGRES_PORT`
-  * `DJANGO_CELERY_BROKER_URL`, using the port from `DOCKER_RABBITMQ_PORT`
-  * `DJANGO_MINIO_STORAGE_ENDPOINT`, using the port from `DOCKER_MINIO_PORT`
-
-Since most of Django's environment variables contain additional content, use the values from
-the appropriate `dev/.env.docker-compose*` file as a baseline for overrides.
-
-## Testing
+## Quick Start
+Using `docker-compose` is the simplest configuration to start with, so it is a prerequisite for this quick start.
+
 ### Initial Setup
-tox is used to execute all tests.
-tox is installed automatically with the `dev` package extra.
+1. Run `docker-compose build`
```

Comment:
Ok I can replace the hyphens with spaces. Should I include a note to explain the difference? If so, how should I write that?

Reply:
I think this is where we would normally have prerequisites: "These installation instructions assume you have a a modern linux based system with a docker version 20.10.13 or greater.  It may work on other systems as well."

Reply:
Thanks. Changed in [9cd2b13](https://github.com/girder/shapeworks-cloud/pull/332/commits/9cd2b135147f6439f1eff44e77c8425f77699c7a)